lib/aggregations/class-custom-date-range.php

```diff
@@ -95,9 +95,12 @@ private function get_date_range( string $from, string $to ) : array {
 		try {
 			$from_datetime = DateTime::createFromFormat( DATE_W3C, $from );
 			$to_datetime   = DateTime::createFromFormat( DATE_W3C, $to );
-			return $from_datetime && $to_datetime
-				? $this->dsl->build_range( $from_datetime, $to_datetime )
-				: [];
+
+			if ( empty( $from_datetime ) && empty( $to_datetime ) ) {
+				return [];
+			}
+
+			return $this->dsl->build_range( $from_datetime, $to_datetime );
```

Comment:
this seems like an unnecessary change -- or am I missing something?

Reply:
No, you are right. Removing it. It is not needed.